### DISCUSSION TITLE
[Esame 17 Luglio 2014] Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#258] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-14 162722](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/50c31b07-8bba-49ae-8801-055151768d8b)

**Comment by cornflexxx** [comment]
$(a)$ Calcolare la probabilità che  ognuno degli $n$ numeri in $\pi$ no sia associato alla propria posizione (Analogo al problema degli accoppiamenti):

$$\mathbb{P}(\text{nessun punto fisso})=1-\mathbb{P}(\text{almeno un punto fisso})$$

Possiamo calcolare il complementare  grazie al PIE:

$$\mathbb{P}(\text{almeno un punto fisso})=\sum \mathbb{P}(\text{1 punto fisso})-\sum \mathbb{P}(\text{2 punti fissi})+...+\sum \mathbb{P}(\text{k punti fissi})+...+\sum \mathbb{P}(\text{n punti fissi})$$

dove $\sum \mathbb{P}(\text{k punti fissi})=\binom{n}{k}\cdot \frac{(n-k)!}{n!}=\frac{1}{k!}$, quindi raggruppando tutte le sommatorie otteniamo : 

$$\mathbb{P}(\text{nessun punto fisso})=1-\mathbb{P}(\text{almeno un punto fisso})=1-1-\frac{1}{2!}+\cdots+\frac{1}{n!}=\sum_{i=0}^{n}\frac{(-1)^i}{i!}$$

$(b)$ Per calcolare esattamente $k$ punti fissi troviamo la probabilità di avere $k$ punti fissi  congiunta a quella che $n-k$ punti non siano fissi , quindi:

$$\mathbb{P}(\text{esattamente k punti fissi})=\binom{n}{k} \cdot \frac{(n-k)!}{n!}\cdot \sum_{i=0}^{n}\frac{(-1)^i}{i!}=\frac{1}{k!}\sum_{i=0}^{n}\frac{(-1)^i}{i!}$$

$(c)$ Calcolo il valore atteso:

$$\mathbb{E}(Z_n)=\sum_{k=0}^{n} k\cdot \frac{1}{k!}\sum_{i=0}^{n}\frac{(-1)^i}{i!}=\sum_{k=1}^{n-1} \frac{1}{(k-1)!}\sum_{i=1}^{n-1}\frac{(-1)^i}{i!}=1$$

$(d)$ Quando $n\to\infty$  abbiamo che : 

$$\sum^{+\infty}\frac{(-1)^i}{i!}=e^{-1}$$

$$\mathbb{P}(Z_n=k)=e^{-1} \cdot \frac{1}{k!}$$

Che è proprio una distribuzione di Poisson con $\lambda =1$
